extensions/interactions/base_test.py

```diff
@@ -33,6 +33,7 @@
 import schema_utils_test
 import utils
 
+FILE = open('abc.txt', 'a')
```

Comment:
What's this used for? If not used, please delete.

Reply:
deleted, thanks.